code/index.html

```diff
@@ -2,17 +2,112 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project Name</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <meta name="viewport" content="width-device-width, initial-scale=1.0">
+    <title>Lolas little treasure box</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="icon" href="Photos/favicon.ico" type="image/icon">
+   
   </head>
   <body>
     <header>
-      <!-- If you want a header with a title, logo and a navbar, here is where you should add that. -->
+      <div class="hamburger">
+          <div></div>
+          <div></div>
+          <div></div>
+      </div>
+      <img src="Photos/Logo.png" alt="Lola">
+      <nav>
+          <a href="#c1"> <strong>Das bin ich, <mark>Lola</mark></strong> </a>
+          <a href="#c2"> Meine kleinen Freuden</a>
+          <a href="#c3">Meine Lieblingswesen</a>
+      </nav>
     </header>
 
-    <main class="">
+<!--Coverbox-->
+<div class="cover">
+  <div class="container">
+  <section class="row">
+    <div class="column">
+      <h1 style="font-size: 300%;">Hi, I'm Lola.</h1>
+      <p style="font-size: 150%; font-weight: 100;">i'm a tiny human on this planet, trying to make sense of this so-called world. <br> 
+        one thought at the time. <br>
+        <br>
+        my biggest achievement up until now: loving the thought of birthdays.
+        <br>
+        i can walk.
+      </p>
+    </div>
+    <div class="column" style="display: flex; flex-direction: row-reverse; " >
+        <img src="Photos/pp.jpg" alt="Lola" style="width: 60%;" >
+    </div>
+</div>
+</section>
+</div>
+</div>
```

Comment:
This whole section should be indented one level since it's part of the `<body>`. Also, it should be indented properly inside (start tag and end tag should be on the same level etc). And again, we should keep the styling in the CSS file! The `<br>` tag can be replaced with some margin/padding on different paragraphs.

```
<div class="cover">
  <div class="container">
    <section class="row">
      <div class="column">
        <h1>Hi, I'm Lola.</h1>
        <p>i'm a tiny human on this planet, trying to make sense of this so-called world.</p>
        <p>one thought at the time.</p>
        <p>my biggest achievement up until now: loving the thought of birthdays.</p>
        <p>i can walk.</p>
      </div>
      <div class="column">
        <img src="Photos/pp.jpg" alt="Lola" style="width: 60%;" >
      </div>
    </section>
  </div>
</div>
```

PS. Like this, it's easier to notice if you have a tag too much, or a tag missing. You had one `<div>` tag too much here :)

Reply:
I have to figure out how to replace br - not quite sure yet